preprocess/download_so.py

```diff
@@ -15,19 +13,6 @@
 else:
     LIBDIR = "/usr/local/lib"
 
-
```

Comment:
GitHub can be extremely slow in some countries, like China, where I live in. This tiny progress indicator can make people know that it does not hang.

Reply:
This process would take on server side, I think don't showing long output of progress could help you at there.
Note that file < 10MB took less than a second on GH Actions.

I encountered its anonymity while watching live ci outputs.

Reply:
We can add it for now, and if it causes any problems, we can revert it any time. it's not really about the main codebase.

Reply:
Sometimes it's needed to run locally, not on github actions, can we put it behind an `if` or something.